app/src/main/java/ch/epfl/sdp/location/LocationService.java

```diff
@@ -36,41 +47,117 @@
     private static final int MIN_UP_INTERVAL_MILLISECS = 1000;
     private static final int MIN_UP_INTERVAL_METERS = 5;
 
+    public static final String ALARM_GOES_OFF = "beeep!";
+
+    public static final String INFECTION_PROBABILITY_TAG = "infectionProbability";
+    public static final String INFECTION_STATUS_TAG = "infectionStatus";
+    public static final String LAST_UPDATED_TAG = "lastUpdated";
+
+    // TODO: This value should be set to several hours. It's now 2 minutes to allow for demo
+    private static long alarmDelayMillis = 120_000;
```

Comment:
What is this all about ?

Reply:
alarmDelayMillis is the (minimum) delay between two successive updates of the infection model